Integrations/SyslogSender/SyslogSender.yml

```diff
@@ -0,0 +1,152 @@
+category: Analytics & SIEM
+commonfields:
+  id: Syslog Sender
+  version: -1
+configuration:
+- display: IP Address (e.g., 127.0.0.1)
+  name: address
+  required: true
+  type: 0
+- defaultvalue: '514'
+  display: Port
+  name: port
+  required: true
```

Comment:
lets not make it mandatory, and in the display name write something like (required for TCP and UDP protocols)